hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetTableLayout.java

```diff
@@ -46,9 +46,9 @@ public TargetTableLayout(final UIEventBus eventBus, final TargetTable targetTabl
         final TargetMetadataPopupLayout targetMetadataPopupLayout = new TargetMetadataPopupLayout(i18n, uiNotification,
                 eventBus, targetManagement, entityFactory, permissionChecker);
         this.eventBus = eventBus;
-        TargetDetails targetDetails = new TargetDetails(i18n, eventBus, permissionChecker, managementUIState, uiNotification,
-                tagManagement, targetManagement, targetMetadataPopupLayout, deploymentManagement, entityFactory,
-                targetTable);
+        TargetDetails targetDetails = new TargetDetails(i18n, eventBus, permissionChecker, managementUIState,
```

Comment:
final TargetDetails

Reply:
Well... if ever acceptable I would prefer not to add `final` on local variables - there is IMHO just not use in it and only makes the code more unreadable.
- Within a method with less than 10 lines lenght it is very unlikely to loose oversight and accidentally reassign a new value to the variable.
- `final` does not mean that the value is constant - but most often this is the whish
- final is a visual clutter in front of the real interesting information: The type of the variable
- The amount of bugs due to accidental reassignment of a local variable is very low (in my life)